website/docs/r/cis_waf_rule.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_waf_rule"
+sidebar_current: "docs-ibm-resource-cis-waf-rule"
+description: |-
+  Provides a IBM CIS WAF Rule Settings resource.
+---
+
+# ibm_cis_waf_rule
+
+Provides a IBM CIS WAF Rule Settings resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to change WAF Rule settings of a domain of a CIS instance.
+
+## Example Usage
+
+```hcl
+resource "ibm_cis_waf_rule" "test" {
+	cis_id     = data.ibm_cis.cis.id
+	domain_id  = data.ibm_cis_domain.cis_domain.id
+	package_id = "c504870194831cd12c3fc0284f294abb"
+	rule_id    = "100000356"
+	mode       = "on"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `cis_id` - (Required,string) The ID of the CIS service instance.
+- `domain_id` - (Required,string) The ID of the domain to change TLS settings.
+- `package_id` - (Required,string) The ID of waf rule package. This field can not be modified.
+- `rule_id` - (Required,string) The ID of waf rule. This field can not be modified.
+- `mode` - (Required,string) The mode setting. This field only once can be set. Valid values: `on`, `off`, `default`, `disable`, `simulate`, `block`, `challenge`.
```

Comment:
Add more description like `The mode to use when the rule is triggered. Value is restricted based on the allowed_modes of the rule`

Reply:
modified